src/components/tiles/Tile.tsx

```diff
@@ -1,75 +1,127 @@
-import React from 'react';
+import React, { FC } from 'react';
 import { Dimensions, TouchableOpacity, View } from 'react-native';
 import { Text } from 'react-native-paper';
 import { useStyles } from '../../hooks/useStyles';
-import { createStyles, Theme } from '../BrandConfigProvider';
+import { createStyles } from '../BrandConfigProvider';
+import TileSelectIcon from './icons/tile-select-chevron.svg';
+import { SvgProps } from 'react-native-svg';
+import LinearGradient from 'react-native-linear-gradient';
 import { tID } from '../../common/testID';
 
-export interface TileProps {
-  onPress?: () => void;
-  id: string;
+interface TileProps {
   title: string;
-  mode?: 'halfLength' | 'fullLength';
+  Icon?: React.FC<SvgProps>;
+  onPress?: () => void;
+  id?: string;
+  testID?: string;
   children?: React.ReactNode;
-  styles?: TileStyles;
 }
 
-export function Tile({
-  onPress,
-  id,
-  title,
-  mode = 'fullLength',
-  children,
-  styles: instanceStyles,
-}: TileProps) {
-  const { styles } = useStyles(defaultStyles, instanceStyles);
+const ViewTile: FC<TileProps> = ({ Icon, title, id, testID }) => {
+  const { styles } = useStyles(defaultStyles);
 
   return (
-    <TouchableOpacity
-      testID={tID(`tile-button-${id}`)}
-      onPress={onPress}
-      disabled={!onPress}
-      style={[
-        mode === 'fullLength'
-          ? { width: tileWidth * 2 + spaceBetweenTiles }
-          : { width: tileWidth },
-        styles.tileColor,
-        styles.tileSize,
-        styles.tileSpacing,
-      ]}
-    >
-      <View testID={tID(`tile-view-${id}`)}>
-        <Text numberOfLines={2} style={styles.label}>
-          {title}
-        </Text>
-      </View>
-      {children}
-    </TouchableOpacity>
+    <View style={styles.container} id={id} testID={testID}>
+      <LinearGradient
+        colors={['transparent', 'transparent']}
+        start={{ x: 0.499, y: -0.2 }}
+        end={{ x: 0.511, y: 1 }}
+        locations={[0, 1]}
+        style={styles.gradient}
+      >
+        <View style={styles.contentsWrapper}>
+          <View style={styles.contents}>
+            {Icon && <Icon style={styles.icon} />}
+            <Text numberOfLines={2} style={styles.titleText}>
+              {title}
+            </Text>
+            <LinearGradient
+              colors={['#509BC5', '#4DC4AF']}
+              start={{ x: -0.76, y: 0 }}
+              end={{ x: 1, y: 0 }}
+              locations={[0, 1]}
+            >
```

Comment:
This feels a little too opinionated, is there a way we could make this configurable or optional if a gradient background is not desired?